<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>modelizerClass</class>
 <widget class="QMainWindow" name="modelizerClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>781</width>
    <height>558</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>modelizer</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>4</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="ModelViewWidget" name="openGLWidget">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="acceptDrops">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="LogWidget" name="logWidget">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>120</height>
       </size>
      </property>
      <property name="documentTitle">
       <string/>
      </property>
      <property name="undoRedoEnabled">
       <bool>false</bool>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="textInteractionFlags">
       <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
      </property>
      <property name="backgroundVisible">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QToolBar" name="toolBar_2">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <property name="iconSize">
    <size>
     <width>12</width>
     <height>12</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>true</bool>
   </attribute>
   <addaction name="actionBlinn"/>
   <addaction name="actionUnlit"/>
   <addaction name="actionWireframe"/>
  </widget>
  <action name="actionImport">
   <property name="icon">
    <iconset resource="modelizer.qrc">
     <normaloff>:/modelizer/Resources/appbar.cabinet.in.png</normaloff>:/modelizer/Resources/appbar.cabinet.in.png</iconset>
   </property>
   <property name="text">
    <string>Import</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionExport">
   <property name="icon">
    <iconset resource="modelizer.qrc">
     <normaloff>:/modelizer/Resources/appbar.cabinet.out.png</normaloff>:/modelizer/Resources/appbar.cabinet.out.png</iconset>
   </property>
   <property name="text">
    <string>Export</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionBatch_convert">
   <property name="icon">
    <iconset resource="modelizer.qrc">
     <normaloff>:/modelizer/Resources/appbar.layer.png</normaloff>:/modelizer/Resources/appbar.layer.png</iconset>
   </property>
   <property name="text">
    <string>Batch convert</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="icon">
    <iconset resource="modelizer.qrc">
     <normaloff>:/modelizer/Resources/appbar.settings.png</normaloff>:/modelizer/Resources/appbar.settings.png</iconset>
   </property>
   <property name="text">
    <string>Settings&lt;</string>
   </property>
  </action>
  <action name="actionBlinn">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="modelizer.qrc">
     <normaloff>:/modelizer/Resources/icon.blinn.png</normaloff>:/modelizer/Resources/icon.blinn.png</iconset>
   </property>
   <property name="text">
    <string>blinn</string>
   </property>
   <property name="shortcut">
    <string>1</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionUnlit">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="modelizer.qrc">
     <normaloff>:/modelizer/Resources/icon.unlit.png</normaloff>:/modelizer/Resources/icon.unlit.png</iconset>
   </property>
   <property name="text">
    <string>unlit</string>
   </property>
   <property name="shortcut">
    <string>2</string>
   </property>
  </action>
  <action name="actionWireframe">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="modelizer.qrc">
     <normaloff>:/modelizer/Resources/icon.wireframe.png</normaloff>:/modelizer/Resources/icon.wireframe.png</iconset>
   </property>
   <property name="text">
    <string>wireframe</string>
   </property>
   <property name="shortcut">
    <string>3</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>Help</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
  <action name="actionAboutQt">
   <property name="text">
    <string>aboutQt</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionOpenHelpMenu">
   <property name="text">
    <string>OpenHelpMenu</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>LogWidget</class>
   <extends>QPlainTextEdit</extends>
   <header>LogWidget.h</header>
  </customwidget>
  <customwidget>
   <class>ModelViewWidget</class>
   <extends>QOpenGLWidget</extends>
   <header>modelviewwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="modelizer.qrc"/>
 </resources>
 <connections/>
</ui>
